components/breadcrumb/src/Crumb.js

```diff
@@ -0,0 +1,59 @@
+/** @jsx jsx */
+
+import React from 'react';
+import PropTypes from 'prop-types';
+import { jsx, useTheme } from '@westpac/core';
+import { ArrowRightIcon } from '../../icon/src';
+import { SrOnly } from '../../accessibility-helpers/src';
+
+// ==============================
+// Component
+// ==============================
+
+/**
+ * Breadcrumb: Breadcrumbs are styled navigational links used to indicate a user journey or path. They are a simple, effective and proven method to aid orientation.
+ */
+export const Crumb = ({ children, last, ...props }) => {
+	const { breadcrumb } = useTheme();
+
+	const common = {
+		display: 'inline-block',
+		position: 'relative',
+		color: breadcrumb.color,
+
+		a: {
+			color: breadcrumb.color,
+			textDecoration: 'none',
+
+			'&:focus, &:hover': {
+				textDecoration: 'underline',
+			},
+		},
+	};
+
+	return (
+		<li css={common} {...props}>
+			{last && <SrOnly>Current page:</SrOnly>}
+			{children}
+			{!last && (
+				<ArrowRightIcon
+					size="small"
+					color={breadcrumb.separator.color}
+					css={{
+						marginLeft: breadcrumb.separator.marginLeft,
+						marginRight: breadcrumb.separator.marginRight,
+					}}
```

Comment:
Should probably add an `aria-hidden="true"` attribute on the icon

Reply:
wondering if we should render these separator icons as background images like with , to better facilitate the non-react system 🤔

TM didn't do that for Tabcordion chevron icon, so maybe they have another solution up their sleeve?

@jaortiz ?

Reply:
I think it's nicer to use the Icon component, going by the whole atomic reuse pattern we are aiming for. I think leave it as the Icon for now and put a note in the issue as to whether we should be doing this as a background image or not

Reply:
We could still use the Icon component, consume its contents and render it out as a background image using the `svgToTinyDataURI` dependency.

Keep it how it is @florettewest. Will mark this for discussion.

Reply:
I updated this question in the issues for breadcrumb...